api/sourcemaps.md

```diff
@@ -0,0 +1,67 @@
+These calls require a project-level access token, which should be provided in the query string. The prefix for all URLs is `https://api.rollbar.com`.
+
+# Upload
+
+To upload your source maps at the beginning of your deploy script (before the new code is in production), call the `api/1/sourcemap` endpoint with the following params:
+
+access_token
+:	a project-level access token
+
+version
+:	a string indicating the current code version
+
+minified_url
+:	the full URL of the minified file. This should start with `http:` or `https:`, which we'll strip off.
+
+source_map
+:	the URL for the source map file
```

Comment:
same comment as about dsym - this needs to be the actual source map file, not the url to it